lib/styles.js

```diff
@@ -89,25 +89,34 @@ module.exports = function (expect) {
             .jsString("'");
     });
 
-    expect.addStyle('key', function (content) {
+    expect.addStyle('key', function (key, isArrayLike) {
         var isSymbol;
         /*jshint ignore:start*/
-        isSymbol = typeof content === 'symbol';
+        isSymbol = typeof key === 'symbol';
         /*jshint ignore:end*/
         if (isSymbol) {
-            this.text('[').sp().appendInspected(content).sp().text(']');
+            this.text('[').sp().appendInspected(key).sp().text(']').text(':');
         } else {
-            content = String(content);
-            if (/^[a-z\$\_][a-z0-9\$\_]*$/i.test(content)) {
-                this.text(content, 'jsKey');
-            } else if (/^(?:0|[1-9][0-9]*)$/.test(content)) {
-                this.jsNumber(content);
+            key = String(key);
+            if (/^[a-z\$\_][a-z0-9\$\_]*$/i.test(key)) {
+                this.text(key, 'jsKey').text(':');
+            } else if (/^(?:0|[1-9][0-9]*)$/.test(key)) {
+                if (!isArrayLike) {
+                    this.jsNumber(key).text(':');
+                }
             } else {
-                this.singleQuotedString(content);
+                this.singleQuotedString(key).text(':');
             }
         }
     });
 
+    expect.addStyle('nonLeadingSpace', function (num) {
+        var lastLine = this.output[this.output.length - 1];
+        if (lastLine.length > 0 && !/\s$/.test(lastLine[lastLine.length - 1].args.content)) {
+            this.sp(num);
+        }
+    });
```

Comment:
I'm a bit worried about this change, it is touching the internal output field on the magicpen, furthermore it makes the assumption that the previous entry was a text entry (I know that it will not fail when content is undefined).